### ISSUE TITLE
Main page has layout issues

### ISSUE THREAD

**Issue by specious** [OpenDataPolicingNC/Traffic-Stops#55]
I noticed that several browsers exhibit the following issues when viewing the [production site](https://opendatapolicingnc.com/):

- Main text overlaps the footer text
- The main text is clipped at the bottom of the page and you cannot scroll down to see rest

I tested with same results on OS X with:

- Firefox Nightly (45.0a1)
- Firefox Developer Edition (43.0a2)
- Firefox (42.0)
- Chrome (48.0.2547.0 dev (64-bit))
- Chrome (43.0.2349.4 canary (64-bit))
- Safari (9.0.1 build 11601.2.7.2)

![main-page-layout-issues](https://cloud.githubusercontent.com/assets/451178/11052087/f6f026b6-8722-11e5-95a0-f4b397352366.png)

**Comment by lotusgraham** [comment]
Yeah, I'm running OSX with updated firefox and Chrome and Safari, all exhibit the same behavior as shown. I'm going to fix it!
